src/tss2-tcti/tctildr.c

```diff
@@ -74,15 +74,23 @@ tcti_from_init(TSS2_TCTI_INIT_FUNC init,
         return TSS2_ESYS_RC_MEMORY;
     }
 
+    /* Keep the level of logging in the loaded tcti low.
+     * It makes too much noise when tcti loader tries them
+     * all one by one and what we want use is the last one.
+     */
+    log_level old_loglevel = LOGMODULE_status;
+    LOGMODULE_status = LOGLEVEL_NONE;
```

Comment:
By doing this, will we ever be able to enable LOGGING inside the init at all anyways ?

Reply:
No, this will quiet all output from any loaded tcti, only tctildr will log messages.

Reply:
And there's no way to turn those warning back on, right ?
So if you want to know why tcti-device failed (e.g.permissions or whatnot) there's no way to get to this info by increasing the debug level, right ?

But I don't have a solution, I just realized that this is the case...

Reply:
How about this code:
```
    log_level old_loglevel = LOGMODULE_status;
    if (LOGMODULE_status <= LOGLEVEL_INFO)
        LOGMODULE_status = LOGLEVEL_NONE;
    r = init(*tcti, &size, conf);
    LOGMODULE_status = old_loglevel;
```

That way we don't silence `TSS2_LOG=tcti+debug` for the tcti's inside.

Reply:
Yeah, that will work better. Updated to comment accordingly.